### DISCUSSION TITLE
Efficiency and speed of multi-core computing

### DISCUSSION THREAD

**Discussion by ZhaofanZhu** [AMReX-Combustion/PeleLMeX#249] [General]
I'm curious when is the most efficient and when is the fastest computation using multi-core computing? I wonder if there is such experience in PeleLMeX, for example, how many grid points each core can process with the highest efficiency?

**Comment by drummerdoc** [comment]
That's going to be highly problem dependent, impacted most by the number of species and stiffness of the chemistry relative to the flow.  Sorry, too complex to answer in this format.

**Reply by ZhaofanZhu** [reply]
Okay, thank you for reply, Mr. Marc. I will do more tests.
